src/index.js

```diff
@@ -132,7 +132,7 @@ function createFolders(folder, options) {
       if (property === 'colour') {
         roofFolder.addColor(options.roof, property);
       } else if (property === 'shape') {
-        const roofTypesAvailable = ['dome', 'flat', 'gabled', 'onion', 'pyramidal', 'skillion', 'hipped', 'round', 'gambrel', 'round'];
+        const roofTypesAvailable = ['dome', 'flat', 'gabled', 'onion', 'pyramidal', 'skillion', 'hipped', 'round', 'gambrel'];
         // If this roof is not supported, add it to the list for sanity.
         if (!roofTypesAvailable.includes(options.roof.shape)) {
           roofTypesAvailable.push(options.roof.shape);
```

Comment:
_(tagged on a line of code to allow for threaded discussion)_

> [Beakerboy](https://github.com/Beakerboy) commented [Jul 18, 2024, 4:28 AM PDT](https://github.com/Beakerboy/OSMBuilding/pull/61#issuecomment-2236268099)
> I submitted a pull request to your branch. Please review it and let me know what you think. I refactored your function to make is shorter and created a unit test for it. It still needs a test for the message in the error log.

I added some feedback to your PR (on my PR) -- https://github.com/justinormont/OSMBuilding/pull/1